lua/nvim-treesitter/indent.lua

```diff
@@ -65,6 +65,7 @@ local indent_funcs = {}
 
 function M.attach(bufnr)
   indent_funcs[bufnr] = vim.bo.indentexpr
+  vim.b.did_indent = 1
   vim.bo.indentexpr = 'nvim_treesitter#indent()'
```

Comment:
maybe set this after the indentexpr assignment ?

Reply:
Still doesnt work on my end